app/controllers/idv/in_person/verify_info_controller.rb

```diff
@@ -77,6 +77,8 @@ def update
           user_id: current_user.id,
           threatmetrix_session_id: flow_session[:threatmetrix_session_id],
           request_ip: request.remote_ip,
+          double_address_verification: current_user.establishing_in_person_enrollment.
+          capture_secondary_id_enabled,
```

Comment:
I would indent this next line more. And since this is the IPP controller, we don't need to add a nil-safe operator on the enrollment, right?

```suggestion
          double_address_verification: current_user.establishing_in_person_enrollment.
            capture_secondary_id_enabled,
```

Reply:
Hmmmm.  We don't use a nil-safe operator in the [in_person verify_step](https://github.com/18F/identity-idp/blob/main/app/services/idv/steps/in_person/verify_step.rb#L31), which made me think we didn't need one here.  But I'll add one just to be safe.